.github/workflows/scripts/models/cpp_graph_inference.sh

```diff
@@ -139,6 +145,7 @@ function main() {
                     NEURAL_SPEED_VERBOSE=1 OMP_NUM_THREADS=$(($cores_per_instance * 1)) numactl -m 0 -C 0-$(($cores_per_instance * 1 - 1)) \
                         $infer_cmd --seed 1234 -t $cores_per_instance -b 2047 -c ${ctx} -n ${output} -m ${model}-${precision}.bin -p "$prompt" 2>&1 | tee ${WORKING_DIR}/${logs_file} || true & 
                     monitor
+                    python ./scripts/cal_acc.py --model_name ${input_model} --init_from_bin ${model}-${precision}.bin --tasks lambada_openai --batch_size 8  2>&1 | tee ${WORKING_DIR}/${logs_file}
```

Comment:
only ”llama" and "gpt-j" support batch_size > 1. We may need a filter.

Reply:
we only use llama and gpt-j in ci